AdobeImsApi/Api/ConfigInterface.php

```diff
@@ -9,7 +9,8 @@
 namespace Magento\AdobeImsApi\Api;
 
 /**
- * Class Config
+ * Declare the the Adobe stock integration config which is responsible for retrieving config
```

Comment:
Adobe Stock should not be mentioned in AdobeIMS module. The keys here are Adobe Creative SDK integration keys